instHWWWW/instHWWWW/Extension/NSMutableAttributedString.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  NS.swift
+//  instHWWWW
+//
+//  Created by Александр Троицкий on 22.10.2022.
+//
+
+import Foundation
+import UIKit
+
+extension NSMutableAttributedString {
+    var fontSize: CGFloat { return 14 }
+    var boldFont: UIFont { return UIFont(
+        name: "AvenirNext-Bold", size: fontSize) ?? UIFont.boldSystemFont(ofSize: fontSize)
```

Comment:
```suggestion
        name: "", size: fontSize) ?? UIFont.boldSystemFont(ofSize: fontSize)
```
Текстовки в константы